pip/README.md

```diff
@@ -41,7 +41,7 @@ BellState()
 
 ## Support
 
-For more documentation and to browse issues, please visit the Q# project wiki at [https://github.com/microsoft/qsharp/wiki].
+For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```

Comment:
Do we want the word "modern" here?

```suggestion
For more information about the Azure Quantum Development Kit, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```

Reply:
a PM might say yes... but I don't feel too strongly. Saves us later updates if/when we want to reduce usage of "Modern"